hexa/files/api.py

```diff
@@ -0,0 +1,183 @@
+import typing
+from dataclasses import dataclass
+
+from django.conf import settings
+from django.core.exceptions import ValidationError
+from google.api_core.exceptions import NotFound
+from google.cloud import storage
+from google.cloud.exceptions import Conflict
+from google.cloud.storage.blob import Blob
+from google.oauth2 import service_account
+
+WORKSPACE_BUCKET_PREFIX = settings.WORKSPACE_BUCKET_PREFIX or ""
```

Comment:
The default `""` could be in `settings.py`, and the dev prefix in the docker compose file